lib/erl_docgen/src/docgen_xmerl_xml_cb.erl

```diff
@@ -81,7 +81,6 @@ convert_tag(ol, Attrs)         -> {list, Attrs};
 convert_tag(strong, Attrs)     -> {em, Attrs};
 convert_tag(td, Attrs)         -> {cell, Attrs};
 convert_tag(tr, Attrs)         -> {row, Attrs};
-convert_tag(tt, Attrs)         -> {c, Attrs};
```

Comment:
I think that this entire module is probably dead code, but anyways xhtml1 allows tt so this module should handle it if this module is ever used for converting xhtml1 to erl_docgen xml....

Reply:
8444a3b613